docs/evm-sidechain/get-started-evm-sidechain.md

```diff
@@ -20,73 +20,59 @@ In order to interact with the network, you need to create an account in the EVM
 
-To connect a MetaMask wallet to the bridge, go to the [EVM sidechain bridge](https://witness-evm-sidechain.peersyst.tech/) and click **Connect Metamask Wallet**.
+To connect a MetaMask wallet to the bridge, go to the [EVM sidechain bridge](https://bridge.devnet.xrpl.org/) and click **Connect Metamask Wallet**.
 
 ![Connect MetaMask Wallet](../img/evm-sidechain-connect-metamask.png '#width=300px;')
 
+### 3. Select a token
+
+Now that both wallets are connected to the bridge, you must select the token you want to transfer. In this case, we are going to select `XRP`.
+
+![Select a token](../img/evm-sidechain-token.png '#width=720;')
 
-### 2. Start the Transaction 
 
-Now that both Xumm and MetaMask wallets are connected to the bridge, you can select the direction of the bridge, the amount, and the address to fund.
+### 4. Start the Transaction 
 
-- **Direction of the bridge**: The direction can be either EVM sidechain → XRP Ledger Devnet or XRP Ledger Devnet → EVM sidechain. Use the **Switch Network** button to switch the direction.
-- **From Amount**: This is the transaction amount. Note that there is a fee to use the bridge.
-    - Network fees: The fees required by the network for your transactions.
-    - Commission: The bridge applies a commission for every transaction completed. This is to prevent spam and distributed denial of service attacks (DDOS).
-- **Destination Address**: The address on the chain where you want the funds.
+With everything configured, it is time to start the bridge transfer. You will now need to enter the amount of XRP that you want to transfer to the EVM Sidechain and click **Transfer** 
 
-    ![Start the transaction](../img/evm-sidechain-initiate-transfer.png '#width=500px;')
+You will be asked to sign transactions on the Metamask wallet and the Faucet wallet.
 
-Enter the details for your transaction and click **Transfer**. Review the details of the transaction carefully before accepting the transaction in the corresponding wallet. 
+![Signature request](../img/evm-sidechain-signature-request.png '#width=720px;')
 
-![Approve the transaction](../img/evm-sidechain-approve-transaction.png '#width=500px;')
+**Note**: The faucet wallet transactions will be signed automatically by the bridge client.
 
-Once you approve the transaction in either the Xumm Wallet or Metamask (depending on the direction of the transfer), a loading screen displays. This process can take up to a few minutes.
+Once you approve the transactions in either the Faucet Wallet or Metamask (depending on the direction of the transfer), a loading screen displays. This process can take up to a few minutes.
 
-![Transaction in progress](../img/evm-sidechain-transfer-in-progress.png '#width=500px;')
+![Transaction in progress](../img/evm-sidechain-transfer-in-progress.png '#width=720px;')
 
 
 ### 3. Receive the Funds
```

Comment:
```suggestion
### 5. Receive the Funds
```

Reply:
Typo fixed! Thank you for the review! Merging this pr should also close #75.